frontend/apps/filemanager/filemanagercollection.lua

```diff
@@ -161,7 +161,7 @@ function FileManagerCollection:showCollDialog()
                     path = G_reader_settings:readSetting("home_dir"),
                     select_directory = false,
                     file_filter = function(file)
-                        return DocumentRegistry:getProviders(file) ~= nil
+                        return DocumentRegistry:hasProvider(file)
```

Comment:
Is the behavior the same for normally hidden files with a custom assigned engine and similar edge cases?

Reply:
Please give an example what do you mean.
We add to favorites only documents (ie they have a provider).

Reply:
I mainly meant I assumed it was a line I wrote a few years back, but I see now that it's in something you wrote. ;-)